src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -101,7 +110,15 @@
 
       <Modal isVisible={isDurationModalShown}>
         <View style={S.parentcolumncontainer}>
-          <Text style={S.setdurationtext}>Set Duration</Text>
+          <View style={S.setdurationview}>
+            <Text style={S.setdurationtext}>Set Duration</Text>
+            {expiration !== 'Never' && (
+              <Text style={S.setdurationdesc}>
+                The poll duration has to be shorter than the expiration time of your post, currently
+                set to {expiration}.
+              </Text>
+            )}
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the text component based on the `expiration` prop is a good practice as it provides more context to the user about the poll duration in relation to the post expiration time.



<!-- This is an auto-generated comment by OSS CodeRabbit -->